packages/cuba-react-core/src/app/MainStore.ts

```diff
@@ -125,10 +125,14 @@ export class MainStore {
 
   @action
   login(login: string, password: string) {
-    return this.cubaREST.login(login, password).then(action(() => {
-      this.userName = login;
-      this.authenticated = true;
-    }))
+    return this.cubaREST.login(login, password)
+      .then(action(() => {
+        this.userName = login;
+        this.authenticated = true;
+      }))
+      .catch(() => console.error(`Login failed. ` +
+        `If your project was generated in CUBA Studio 14+, please make sure that the correct ` +
+        `REST API Client ID and Secret are set in environment variables.`))
```

Comment:
I think it is redundant to show such message on **any** error (e.g. it will be shown on wrong credentials). We can show it only if we can determine that it has been caused by wrong secret/client and in DEVELOPMENT build only.

Reply:
Not sure we can determine error cause. Removed.